src/DynamoCore/Search/SearchDictionary.cs

```diff
@@ -13,6 +13,7 @@ namespace Dynamo.Search
     public class SearchDictionary<V>
     {
         private ILogger logger;
+        private static int LIMIT_SEARCH_TAG_SIZE = 100;
```

Comment:
could this be too small for some localized searches?

Reply:
Maybe...the value is definitely up for debate. I wonder if it would actually be best to warn the user if this limit is surpassed (I am not sure when or where)
I can take a look at the tags we got from Jostein to get a better idea of how many characters there are in a "normal" situation

Reply:
Increased to 300